test/multi_line_string_spec.rb

```diff
@@ -0,0 +1,78 @@
+require_relative './helper'
+
+describe Terraformer::MultiLineString do
+
+  describe 'construction' do
+
+    it 'constructs from coordinates' do
+      a = Terraformer::Coordinate.new -122.6764, 45.5165
+      b = a + [0.02, 0.02]
+      c = b + [0.1, -0.1]
+      mls = Terraformer::MultiLineString.new a, b, c
+      mls.to_json.must_equal '{"type":"MultiLineString","coordinates":[[[-122.6764,45.5165],[-122.6564,45.5365],[-122.5564,45.4365]]]}'
+      mls.must_be_valid_geojson
+    end
+
+    it 'constructs from array - single line' do
+      ## unlike point, multipoint and linestring, passing in the coordinates array directly into multilinestring doesn't work
```

Comment:
MultiLineString is the only class that cannot be initialized with the output of the "coordinates" block from the corresponding geojson. I think this is a bug and should be addressed. I have most of a fix, but I think it should be part of a different pull request.